contrib/openstack/README.md

```diff
@@ -1,3 +1,16 @@
-# Provision a Deis Cluster on OpenStack
+# Provision a Deis Cluster on Openstack
 
 Please refer to the instructions at http://docs.deis.io/en/latest/installing_deis/openstack/.
+
+Meanwhile here is how I've been testing it:
```

Comment:
This sounds too informal even for our contrib/ docs--could you rephrase it as something like `Provision a cluster with a command such as:`